cli/shared/src/main/scala/nativetoolscli/options/ConfigOptions.scala

```diff
@@ -0,0 +1,27 @@
+package scalanativecli.options
+
+import caseapp._
+
+// format: off
+case class ConfigOptions(
+
+    @Group("Config")
+    @HelpMessage("Required path of the resulting output binary.")
+    @ValueDescription("<output-path>")
+    outpath: String,
```

Comment:
We can also define here additonal short name `-o`, i think it can be done with `@Name` of `@ExtraName` annotaiton